build.gradle

```diff
@@ -38,9 +38,11 @@ repositories {
 }
 
 dependencies {
-	minecraft "com.mojang:minecraft:1.14.4"
-	mappings "net.fabricmc:yarn:1.14.4+build.1"
+	// TODO: Stop depending on Minecraft in main build.gradle
+	minecraft "com.mojang:minecraft:1.16.3"
+	mappings "net.fabricmc:yarn:1.16.3+build.27:v2"
 
+	// TODO: Jetbrains annotations
```

Comment:
actually, since we are using the annotations in fapi now, would this now be able to use the annotations as well

Reply:
We can't depend on fabric api. Annotations are for a later date but I don't like how discussions are going just for annotations in fabric loader

Reply:
I took fapi as an example, the jetbrains annotations are applied through loom 0.5, I do admit that it doesn't have much to do with this PR tho.

Reply:
I will look into fixing latest loom as a loader dev env, I dont see an issue with using JB's annotations in loader?